.changeset/strong-weeks-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": patch
+---
+
+[Fixed] - Issue where mobile navigation toggle wouldn't close.
```

Comment:
let's log the other changes as well

Reply:
Given that all the other changes are internal to the repo and all test specific, I didn't add these as they don't warrant a version bump, and as a result, shouldn't be added to the changeset, as it has no impact on users 👍🏼

Reply:
Shouldn't they go in the root changelog though? (not sure if we have changesets there yet, but they should be logged in there)

Reply:
Ah yeah, root is a good place :)